assets/css/sidebar.css

```diff
@@ -1,8 +1,10 @@
 .sidebar {
+  --sidebarFontSize: 16px;
+  --sidebarLineHeight: 20px;
   font-family: var(--sansFontFamily);
-  font-size: 16px;
+  font-size: var(--sidebarFontSize);
   font-weight: 300;
-  line-height: 18px;
+  line-height: var(--sidebarLineHeight);
```

Comment:
This technically changes the value from 18px to 20px but I didn't notice any visual changes. I think all possible children here overwrite the line-height to 20px anyway.